gsuite_reports_rules/gsuite_google_access.py

```diff
@@ -0,0 +1,10 @@
+def rule(event):
```

Comment:
Couldn't think of a good dedup/title function here, probably doesn't need one. The event is indicating that a google employee is looking at your documents for some reason. There are more details in the gsuite super admin console.